File: module-music/src/main/java/com/yyxnb/module_music/adapter/MusicLocalListAdapter.java
package com.yyxnb.module_music.adapter;


import com.yyxnb.what.adapter.base.BaseAdapter;
import com.yyxnb.what.adapter.base.BaseViewHolder;
import com.yyxnb.module_music.R;
import com.yyxnb.module_music.bean.MusicLocalBean;

public class MusicLocalListAdapter extends BaseAdapter<MusicLocalBean> {

    public MusicLocalListAdapter() {
        super(R.layout.item_music_home_list);
    }

    @Override
    protected void bind(BaseViewHolder holder, MusicLocalBean item, int position) {
        holder.setText(R.id.tvTitle, item.title)
                .setText(R.id.tv_anchor, item.author);
    }


}

File: module_music/src/main/java/com/yyxnb/module_music/adapter/MusicLocalListAdapter.kt
package com.yyxnb.module_music.adapter

import com.yyxnb.adapter.BaseAdapter
import com.yyxnb.adapter.BaseViewHolder
import com.yyxnb.module_music.R
import com.yyxnb.module_music.bean.MusicLocalBean

class MusicLocalListAdapter : BaseAdapter<MusicLocalBean>(R.layout.item_music_home_list) {

    protected override fun bind(holder: BaseViewHolder, item: MusicLocalBean, position: Int) {
        holder.setText(R.id.tv_title, item.title)
                .setText(R.id.tv_anchor, item.author)
    }
}